src/reference/modules/installation/pages/_partials/initial_config.adoc

```diff
@@ -1,42 +1,8 @@
 == Initial configuration
 
-After the installation, you have to configure some system elements, by launching
-the following initialisation script:
-
-----
-
-/opt/rudder/bin/rudder-init
-
-----
-
-This script will ask you to fill in the following details:
-
-=====
-
-Allowed networks:: A list of IP networks authorized to connect to the server.
-It uses the network/CIDR mask notation, for instance +192.168.0.0/24+ or
-+10.0.0.0/8+.
-To add several networks, first type the first network, then press
-the return key - the script will ask if you wish to add some more networks.
-Also, the allowed networks can be adjusted later in the web interface
-in the Administration - Settings tab without having to run the script
-again.
-
-=====
-
-[TIP]
-
-=====
-
-In case of typing error, or if you wish to reconfigure Rudder, you can
-execute this script again as many times as you want.
-
-=====
-
 === Validate the installation
```

Comment:
Could you add a line in this section advising to go to `Settings -> General -> Allowed Networks` and check and update the list of networks?